Rename Js(Copy|Create)PropertyIdUtf8 to Js(Copy|Create)PropertyId

xiaoyinl opened the pull request:
There is no exported function called JsCopyPropertyIdUtf8, but there is JsCopyPropertyId,
[declared](https://github.com/Microsoft/ChakraCore/blob/6ec963b359d7c515a1bb05b5ffc169fd321e28a7/lib/Jsrt/ChakraCore.h#L446) in ChakraCore.h. So update the doc to match the source code.

xiaoyinl committed:
Rename JsCopyPropertyIdUtf8 to JsCopyPropertyId

There is no exported function called JsCopyPropertyIdUtf8, but there is JsCopyPropertyId,
declared in ChakraCore.h. So update the doc to match the source code.

msftclas commented:
@xiaoyinl,
Thanks for having already signed the Contribution License Agreement. Your agreement was validated by Microsoft. We will now review your pull request.
_Thanks,
Microsoft Pull Request Bot_

xiaoyinl committed:
Rename JsCreatePropertyIdUtf8 to JsCreatePropertyId

There is no exported function called JsCreatePropertyIdUtf8, but there is JsCreatePropertyId,
declared in ChakraCore.h. So update the doc to match the source code.

liminzhu reviewed (approved):
LGTM. Thanks!
